src/main/docker/gitlab-gitlabci.yml

```
@@ -0,0 +1,37 @@
+version: "3.6"
+services:
+    gitlab:
+        build: gitlab
```

```suggestion
        build: gitlab
        container_name: gitlab
```

This allows easy referencing (See readme comment)